tests/cfncluster-release-check.py

```diff
@@ -66,7 +69,7 @@ def run_test(region, distro, scheduler):
     file.write("aws_region_name = %s\n" % region)
     file.write("[cluster default]\n")
     file.write("vpc_settings = public\n")
-    file.write("key_name = default\n")
+    file.write("key_name = cfncluster-test\n")
```

Comment:
Doh; this was me being lazy.  Let's make this a command line argument instead of just changing the name.